src/language.py

```diff
@@ -31,6 +31,7 @@ class Language(object):
   EN = _Language('English', 'en', 'US')
   ES = _Language('Español', 'es', 'ES')
   FR = _Language('Français', 'fr', 'FR')
+  ZH_CN = _Language('简体中文', 'zh_CN', 'CN')
```

Comment:
```suggestion
  ZH_CN = _Language('简体中文', 'zh-CN', 'CN')
```
Note: only update the second instance as Python dictionary names must not have dashes in them.